setup.cfg

```diff
@@ -1,3 +1,8 @@
+[pytest]
+addopts = --cov=hera -vv
+# for now, keep this commented out, as it's used explicitly in Makefile
+; testpaths = tests
+
```

Comment:
This configuration can be put on tox.ini. 
ref: https://docs.pytest.org/en/6.2.x/customize.html#tox-ini

I prefer the test related configurations is placed in tox.ini. What do you think about this?

Reply:
Hi @hirosassa !

I keep this in `setup.cfg` so when one run's the tests **not** using `tox`, he uses the same `pytest` configuration.

For example - 

```bash
> make test
> pytest ./tests/[specific-test]
```

Meaning - I want to keep `tox` for the purpose of isolated builds, not for the source of all of the configuration.
I'm aware of other projects [ also `luigi` ] using it to _also_ to replace `setup.cfg`. I just don't like it :) 

acceptable ?

Reply:
Yeah, I understand what you intended 👍 I agree with this.

> I want to keep tox for the purpose of isolated builds, not for the source of all of the configuration.

Just for your information, you can write configuration of `pytest` on tox.ini without using `tox`.

ref:
https://github.com/pytest-dev/pytest/blob/e86bf7fda0a73aebee7d3741e1106d58cf8dec85/src/_pytest/config/findpaths.py#L88-L111

Reply:
I was not aware `tox.ini` is a part of the [defaults configs list](https://github.com/pytest-dev/pytest/blob/e86bf7fda0a73aebee7d3741e1106d58cf8dec85/src/_pytest/config/findpaths.py#L93) !

The question now rises regarding other tools I guess.. 

Do you think it's worth the effort to check with the rest of the tools if they behave the same ?

I see [luigi](https://github.com/spotify/luigi/blob/master/tox.ini) indeed got rid of `setup.cfg` all together in favour of `tox` ..